### DISCUSSION TITLE
The stroke is not working for any geometry

### DISCUSSION THREAD

**Discussion by Shafagh777** [3b1b/manim#1455] [Q&A] [answered]
I ran the following code and it does not drawing the stroke line:

```
from manimlib import *
class shape(Scene):
    def construct(self):
                  circle=Circle()
                  self.add(circle)
```

would be glad to have your comments on this issue. I have also tried the following code:

```
from manimlib import *

class SquareToCircle(Scene):
    def construct(self):
        circle = Circle()
        circle.set_fill(BLUE, opacity=0.5)
        circle.set_stroke(BLUE_E, width=4)

        self.add(circle)
```

and the stroke is not acting.

**Comment by rolisz** [comment]
I just ran your second example and I can see the stroke with a different shade of blue. Or is the question about having it appear animated?

**Reply by fabriciosb** [reply]
For me it does not show any strokes either! Could be some kind of configuration or problem with the instalation or drives for openGL?

**Comment by Shafagh777** [comment]
@rolisz I have tried in different PC and not showing the stroke. I have also checked the version of open GL which was 4.6. I think it is a bug in the manimgl. Should I change any config somewhere in manimlib?

**Comment by Shafagh777** [comment]
@fabriciosb you need to have OpenGL installed higher than version 3.3. although mine is 4.6 and still have the same problem. I truly can not figure it out.

**Reply by fabriciosb** [reply]
Mine is 4.6 as well. No clue what is the problem. I also have installed the community version of Manim and it works fine. But I wanted to try the realtime features of manimgl.

**Comment by niedong** [comment, marked as answer]
Hi, I've solved this problem. Please change your graphics processor. I changed my graphics processor and everything becomes fine.

vendor: NVIDIA Corporation
renderer: GeForce 940MX/PCIe/SSE2
version: 3.3.0 NVIDIA 461.40
python: 3.9.4 (tags/v3.9.4:1f2e308, Apr  6 2021, 13:40:21) [MSC v.1928 64 bit (AMD64)]
platform: win32
code: 330
